HW#6.1.py

```diff
@@ -0,0 +1,28 @@
+# Time
+n = 808
+hours, minutes = n // 60, n % 60
+print(hours, minutes)
+result = hours // 10 + hours % 10 + minutes // 10 + minutes % 10
+print(result)
+
+# Level Up
+experience = 10
+threshold = 15
+reward = 5
+if (experience + reward) >= threshold:
+    print("true")
+else:
+    print("false")
+
+# Time converter
```

Comment:
Better to convert to `int` only once, otherwise you are doing the same operation